packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -3987,9 +3987,9 @@ describe('mouse state', () => {
           clientX: 492,
           clientY: 9,
           layerX: 492,
-          layerY: 215,
+          layerY: 226,
```

Comment:
```suggestion
          layerY: 215,
```

Reply:
why was this changed?

Reply:
The test failed for me before but now it works so I changed it back